swagger.yaml

```diff
@@ -374,6 +470,25 @@ components:
         - MONTH
         - WEEK
 
+    VacationType:
```

Comment:
Very minor detail but could it be worth having all the vacation related schemas grouped together rather than VacationType and RequestStatus here and Vacation Request below?